src/styles/utils/variables.scss

```diff
@@ -0,0 +1,6 @@
+$stopwatch-size: 80vmin;
+$center-size: 5vmin;
+$minutes-hand-hight: 20vmin + $center-size / 2;
```

Comment:
It seems there is a typo in the variable name `$minutes-hand-hight`. According to the standard naming conventions, it should be `$minutes-hand-height`.